src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -186,6 +186,7 @@
             <argument type="service" id="sylius.repository.order" />
             <argument type="service" id="sm.factory" />
             <argument>%sylius_order.order_expiration_period%</argument>
+            <argument type="service" id="monolog.logger" />
```

Comment:
Isn't there a service just called `logger`?

Reply:
Done. `logger` is an alias for "monolog.logger", so I think it wont make any difference